src/android/CHIPTool/build.gradle

```diff
@@ -21,6 +21,13 @@ allprojects {
     }
 }
 
+ext {
+    // true to build SDK from source code and debug in Android Stduio
+    // use scripts/examples/android_app_as.sh to preapre it
+    // false to use SDK prebuilt by  scripts/examples/android_app.sh
+    matterFromSource = false
```

Comment:
I would prefer to somehow combine this on top of or with #8862 and reduce the paths used to compile things. I imagine we want a single 'IDE build' and  a single 'build via script'. For the 'build via script' I would like to use 'build_example.py' since that goes into the gcp automated build as well.

Reply:
Agree on single build via script, but within the IDE it's nice to have these two paths. CHIP devs probably always want `matterFromSource = true`, but eventually clients integrating with the Android controller will just add a gradle dep for it, and not build it from source.

Reply:
Yes, I`d like to add a "IDE build". I will have a look at https://github.com/project-chip/connectedhomeip/pull/8862